samples/event_manager/src/modules/stats.c

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+
+#define MODULE stats
+
+#include "measurement_event.h"
+
+#include <logging/log.h>
+#define STATS_LOG_LEVEL 4
+LOG_MODULE_REGISTER(MODULE, STATS_LOG_LEVEL);
+
+#define STATS_ARR_SIZE 10
+
+static s32_t val_arr[STATS_ARR_SIZE];
+
+static void add_to_stats(s32_t value)
+{
+	static size_t ind;
+
+	/* Add new value */
+	val_arr[ind] = value;
+	ind++;
+
+	if (ind >= ARRAY_SIZE(val_arr)) {
+		ind = 0;
+		s64_t sum = 0;
+
+		for (size_t i = 0; i < ARRAY_SIZE(val_arr); i++) {
+			sum += val_arr[i];
+		}
+		LOG_INF("Average value3: %d", (s32_t)(sum/ARRAY_SIZE(val_arr)));
```

Comment:
`%d` is int not `s32_t`...
if you want to cast to s32_t you would have to use special formatting string (see `PRId32` in `inttypes.h`). I recomend staying with int

Reply:
ok, changed to int